GoogleForCreators/web-stories-wp
# Programmatically migrate templates

## Summary



## Relevant Technical Choices

Uses Rollup to create an ES bundle for use with the CLI script, as webpack does not support ES module output.

## To-do

* [x] Tests for `bin/utils/updateTemplates.js`

## User-facing changes



## Testing Instructions

1. Build the migration script by running `npx rollup --config bin/rollup.config.js`
1. Manually lower the version number of one of the templates
1. Run `npm run workflow:templates`
1. Verify that templatee has been migrated to the current version
1. Verify that the `version` key of the template object is at the top of the file.

---



See #2120
